src/trace_processor/trace_processor_shell.cc

```diff
@@ -774,11 +774,13 @@ General purpose:
                                       If used with --run-metrics, the query is
                                       executed after the selected metrics and
                                       the metrics output is suppressed.
- --add-sql-module PACKAGE_PATH         Files from the directory will be treated
+ --add-sql-module PACKAGE_PATH
+ --add-sql-package PACKAGE_PATH       Files from the directory will be treated
                                       as a new SQL package and can be used for
                                       INCLUDE PERFETTO MODULE statements. The
                                       name of the directory is the module name.
- --override-sql-module PACKAGE_PATH   Will override trace processor package with
+ --override-sql-module PACKAGE_PATH
```

Comment:
Can you please move these to the Advanced section and say they are the same as --add-sql-package/--override-sql-package but with more acccurate names and are only kept around for backwards compatbility?

Reply:
Done.